repotracker/repotracker.go

```diff
@@ -894,13 +894,22 @@ func createVersionItems(ctx context.Context, v *model.Version, metadata model.Ve
 			}
 			for _, t := range buildvariant.Tasks {
 				var match bool
-				match, err = aliasesMatchingVariant.HasMatchingTask(projectInfo.Project.FindProjectTask(t.Name))
+				name, tags, ok := projectInfo.Project.GetTaskNameAndTags(t)
```

Comment:
Similar confusion here - assuming that this loop can include task groups, at the bottom we would create a TVPair for an entire task group. I don't think that works does it? I must be missing something